jpo-ode-core/src/main/java/us/dot/its/jpo/ode/model/OdeTimPayload.java

```diff
@@ -26,7 +26,7 @@
  * ODE TIM payload class for both J2735 TravelerInformation and ODE 
  * TIM Creator OdeTravelerInformationMessage.
  */
-public class OdeTimPayload extends OdeMsgPayload {
+public class OdeTimPayload extends OdeMsgPayload<OdeObject> {
```

Comment:
Updating classes that inherit OdeMsgPayload to now explicitly provide the OdeObject class as the type. This used to be the default, but had to change to allow for using ASN POJOs as part of the payload. Other check style changes were also applied.